.evergreen/config/functions.yml

```diff
@@ -192,17 +192,23 @@ functions:
           ${PREPARE_SHELL}
           bash ${DRIVERS_TOOLS}/.evergreen/stop-orchestration.sh
 
-  "run OCSP responder":
-    - command: shell.exec
+  "start OCSP responder":
+    - command: subprocess.exec
       params:
+        binary: bash
+        background: true
         include_expansions_in_env:
-          - TEST_COLUMN
-          - CERT_TYPE
-          - USE_DELEGATE
```

Comment:
Noted that these expansions ere used to determine what is now passed in `SERVER_TYPE` and `OCSP_ALGORITHM`.